pallets/loans/src/migrations.rs

```diff
@@ -87,6 +120,15 @@ pub mod v3 {
                 })
             });
 
+            MarketRewardSpeed::<T>::iter().for_each(|(asset_id, reward_speed)| {
+                RewardSupplySpeed::<T>::insert(asset_id, reward_speed);
+                RewardBorrowSpeed::<T>::insert(asset_id, reward_speed);
+            });
+
+            RewardAccured::<T>::iter().for_each(|(account_id, reward_amount)| {
```

Comment:
need remove old storage after the migration

Reply:
👌🏻